lib/client.js

```diff
@@ -7,8 +7,21 @@ var OKCupid = function(){
 }
 
 OKCupid.prototype.login = function(username, password, callback){
-    var login_form = {username: username, password: password};
-    this.requester.postRequest(URLS.login, login_form, callback)
+    var login_form = {
+      okc_api: 1, 
+      username: username, 
+      password: password,
+    };
+    this.requester.postRequest(URLS.login, login_form, function(err, res, body) {
+      // if there is a non-zero status in response and response does not show an error
+      if (body.status !== 0 && !err) {
```

Comment:
Ahh great catch. This is gotcha that would have taken me a little while to figure out. Thank you for the clarification. I just re-read the section on short-circuit evaluation on MDN. I will commit the change here shortly.